mariadb103.spec

```diff
@@ -116,7 +116,7 @@
 %bcond_without unbundled_pcre
 %else
 %bcond_with unbundled_pcre
-%global pcre_bundled_version 8.43
+%global pcre_bundled_version 10.33
```

Comment:
The bundled PCRE version is defined upstream in [pcre/configure.ac](https://github.com/MariaDB/server/blob/mariadb-10.3.18/pcre/configure.ac#L11-L12).  It is still 8.43 at the mariadb-10.3.18 tag, so we can keep this line the same.